tokenList.json

```diff
@@ -356,7 +369,7 @@
   },
   "NEX": {
     "symbol": "NEX",
-    "companyName": "Neon Exchange",
+    "companyName": "NEX Token",
```

Comment:
@comountainclimber @dvdschwrtz It looks like the generated token doesn't match what you guys manually put in here as part of #34.  Was that value manually massaged as part of that PR?  I imagine that if/when the automated CI builds are turned back on, this value will be overwritten as "NEX Token" going forward.

Reply:
the generated token name looks alright 👍